### DISCUSSION TITLE
Sort articles by oldest first

### DISCUSSION THREAD

**Discussion by PhilC813** [jocmp/capyreader#370] [Feature Requests]
Hello again! I don't think this has been suggested yet. I was wondering if we could have the equivalent of this option available in the Feedbin website settings: 
![Screenshot_20240917_195428.jpg](https://github.com/user-attachments/assets/7d1659ff-378d-4cb2-bb3b-34cb36b79d09)

It's especially useful for news websites, as you may want to see what you've missed starting from the oldest unread news. 

Thank you!

**Comment by jocmp** [comment]
hey again @Philippe-Choquette - thanks for the suggestion. I tried this out on my Feedbin account and it seems pretty simple to add. I'll try and add it in the next week or two. https://github.com/jocmp/capyreader/issues/382

**Reply by PhilC813** [reply]
Thank you so much!

**Comment by PhilC813** [comment]
I've tried the GitHub version "2024.09.1053-dev" in which you added the feature and it works perfectly. Thank you for taking the input!

**Comment by jocmp** [comment]
Closing now that the feature has been added.

**Reply by PhilC813** [reply]
Hey Josiah!

When you added this setting back in '24, you implemented it as it is on Feedbin's website (ircc), named "Unread sort". Since then, it has been renamed as "Article sort", and so it takes effect on both read and unread articles.

With the previous implementation, I could have unread articles sorted by "oldest first", and read articles stayed sorted by "newest first", just the way I like it. This is not possible anymore.

Do you think there could be a separate sort option for read and unread articles maybe?

Thank you!!

**Reply by PhilC813** [reply]
@jocmp in case you missed my comment above
